jena-base/src/main/java/org/apache/jena/atlas/iterator/Iter.java

```diff
@@ -460,6 +475,14 @@ public T next() {
             return t;
         }
 
+        @Override
+        public void forEachRemaining(Consumer<? super T> action) {
+            stream.forEachRemaining(item->{
+                this.action.accept(item);
```

Comment:
Two calls to action?

Reply:
Indeed. org.apache.jena.atlas.iterator.Iter.IterOperate is initialised with a `Consumer<T> action`. 
When consuming via org.apache.jena.atlas.iterator.Iter.IterOperate#hasNext, the consumer is also called, so when consuming via `org.apache.jena.atlas.iterator.Iter.IterOperate#forEachRemaining` there are two consumers, the one the class was initialised with and the one that is a parameter of the method.